.editorconfig

```diff
@@ -0,0 +1,9 @@
+root=true
+
+[*.pl]
+indent_style = space
+indent_size = 2
+
+[*.py]
```

Comment:
I have since (many years ago) come to realise that space/2 is right for Python. I'd prefer to fix the .py file that way and make `.editorconfig` reflect that :)

Reply:
done!